packages/vite/src/node/__tests__/utils.spec.ts

```diff
@@ -270,3 +272,15 @@ describe('isFileReadable', () => {
     })
   }
 })
+
+describe('processSrcSetSync', () => {
+  test('replace srcset with base url /base/', async () => {
```

Comment:
I might word this as:
```suggestion
  test('prepend base URL to srcset', async () => {
```